akvo/rsr/models/organisation.py

```diff
@@ -175,6 +175,9 @@ def org_type_from_iati_type(cls, iati_type):
         'self', null=True, blank=True, on_delete=models.SET_NULL,
         help_text=_(u'Organisation that maintains content for this organisation through the API.')
     )
+    # pointer to original org if this is a shadow. Used by EUTF
```

Comment:
I think the comment here could go into the `help_text`

Reply:
Heh, I know I thought about that when I wasn't at the computer. Then I forgot to put it there...